src/components/UserStatistics/index.jsx

```diff
@@ -69,7 +70,7 @@ class AccountStatistics extends Component {
             if (newFollowers > 0) {
               newFollowers = `+${newFollowers}`
             } else if (newFollowers < 0) {
-              newFollowers = `-${newFollowers}`
+              newFollowers = `${newFollowers}`
```

Comment:
The number is already negative. In the data table it shows up with two minus signs `--` and that brakes the graph. So just remove it to makes things work correctly